python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,131 @@
+plugin:
+  name: Generate PlantUML Diagrams
+  package: aac.plugins.puml
+  description: |
+    An AaC plugin that enables PlantUML diagrams to be generated from AaC
+    definitions.
+  commands:
+    - name: puml-component
+      help_text: Converts an AaC-defined system to a PlantUML component diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML component diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Component Diagram Test
+        scenarios:
+          - name: Output PlantUML component diagram for valid architecture.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-component command.
+            then:
+              - A PlantUML component diagram of the model is written to the output directory.
```

Comment:
If you are wanting to repurpose this scenario, then it needs more than just the result modified. As it is right now, it makes it seem that you can provide only the architecture file and get a result, and that is incorrect. 

```suggestion
          - name: Output PlantUML component diagram to the requested output directory.
            given:
              - The {{puml.input.architecture-file}} contains a valid model.
              - The positional argument {{puml.input.output-directory}} contains a valid file path.
            when:
              - The aac app is run with the puml-component command.
            then:
              - A PlantUML component diagram of the model is written to the output directory.
```